lib/zone-spec/async-test.ts

```diff
@@ -1,3 +1,5 @@
+import { Zone, ZoneSpec, Task, ZoneDelegate, HasTaskState } from '../zone';
```

Comment:
We can not add this code here. If we do then Zones will be included in the `async-test.js` bundle as well as `zone.js` bundle. 

Zone's is meant as a polyfil, and so it should be ambiently available.